common/src/main/java/com/microsoft/identity/common/internal/broker/MicrosoftAuthClient.java

```diff
@@ -106,6 +111,19 @@ private Intent getIntentForAuthService(final Context context) {
         return authServiceToBind;
     }
 
+    private boolean isMicrosoftAuthServiceSupported(@NonNull final Context context, @NonNull final String packageName) {
+        if (packageName == null || packageName.length() == 0) {
+            return false;
+        }
+        final Intent microsoftAuthServiceIntent = new Intent(MICROSOFT_AUTH_SERVICE_INTENT_FILTER);
+        microsoftAuthServiceIntent.setPackage(packageName);
+        microsoftAuthServiceIntent.setClassName(packageName, MICROSOFT_AUTH_SERVICE_CLASS_NAME);
+
+        final PackageManager packageManager = context.getPackageManager();
```

Comment:
[If all you need is the `PackageManager` we should pass _that_ in, instead of the whole `Context`.](https://en.wikipedia.org/wiki/Principle_of_least_privilege)